src/components/TodoList/TodoList.tsx

```diff
@@ -1,210 +1,117 @@
-            <td className="is-vcentered"> </td>
-
-            <td className="is-vcentered is-expanded">
-              <p className="has-text-danger">fugiat veniam minus</p>
-            </td>
-
-            <td className="has-text-right is-vcentered">
-              <button data-cy="selectButton" className="button" type="button">
-                <span className="icon">
-                  <i className="far fa-eye-slash" />
-                </span>
-              </button>
-            </td>
-          </tr>
-
-          <tr data-cy="todo">
-            <td className="is-vcentered">4</td>
-            <td className="is-vcentered"><span className="icon" data-cy="iconCompleted"><i className="fas fa-check" /></span></td>
-            <td className="is-vcentered is-expanded">
-              <p className="has-text-success">et porro tempora</p>
-            </td>
+  const dispatch = useDispatch();
+  const todos = useAppSelector(state => state.todos);
+  const currentTodo = useAppSelector(state => state.currentTodo);
+  const {
+    query,
+    status,
+  } = useAppSelector<{ query: string, status: Status }>(state => state.filter);
 
-            <td className="has-text-right is-vcentered">
-              <button data-cy="selectButton" className="button" type="button">
-                <span className="icon">
-                  <i className="far fa-eye" />
-                </span>
-              </button>
-            </td>
-          </tr>
+  const selectTodo = (todo: Todo) => dispatch(currentTodoActions.setTodo(todo));
 
-          <tr data-cy="todo">
-            <td className="is-vcentered">5</td>
-            <td className="is-vcentered"> </td>
+  const getFilteredTodos = () => {
+    let filteredTodos = todos;
 
-            <td className="is-vcentered is-expanded">
-              <p className="has-text-danger">laboriosam mollitia et enim quasi adipisci quia provident illum</p>
-            </td>
+    switch (status) {
+      case Status.active:
+        filteredTodos = todos.filter(todo => !todo.completed);
+        break;
 
-            <td className="has-text-right is-vcentered">
-              <button data-cy="selectButton" className="button" type="button">
-                <span className="icon">
-                  <i className="far fa-eye" />
-                </span>
-              </button>
-            </td>
-          </tr>
+      case Status.completed:
+        filteredTodos = todos.filter(todo => todo.completed);
+        break;
 
-          <tr data-cy="todo">
-            <td className="is-vcentered">6</td>
-            <td className="is-vcentered"> </td>
+      case Status.all:
+      default:
+        filteredTodos = todos;
+    }
 
-            <td className="is-vcentered is-expanded">
-              <p className="has-text-danger">qui ullam ratione quibusdam voluptatem quia omnis</p>
-            </td>
+    return filteredTodos.filter(todo => {
+      return todo.title.toLowerCase().includes(query.toLowerCase());
+    });
```

Comment:
It's more readable, i think
up to you
```suggestion
    return filteredTodos.filter(todo => {
      const lowerCaseTitle = todo.title.toLowerCase();
      const lowerCaseQuery = todo.title.toLowerCase();
      
      return lowerCaseTitle.includes(lowerCaseQuery);
    });
```